manim/mobject/types/vectorized_mobject.py

```diff
@@ -906,6 +906,181 @@ def __init__(self, *vmobjects, **kwargs):
         VMobject.__init__(self, **kwargs)
         self.add(*vmobjects)
 
+class VDict(VMobject):
+    '''A VGroup-like class, also offering submobject access by
+    key, like a python dict
+
+    Parameters
+    ----------
+    pairs: Tuple[Hashable, :class:`~.VMobject`]
+            Each pair is a 2-element :class:`tuple` wherein the first 
+            element is the key for the mobject and the second
+            element is the actual mobject
+    show_keys : :class:`bool`, optional
+            Whether to also display the key associated with 
+            the mobject. This might be useful when debugging,
+            especially when there are a lot of mobjects in the
+            :class:`VDict`. Defaults to False
+    kwargs : Any
+            Other arguments to be passed to `Mobject` or the CONFIG.
+
+    Attributes
+    ----------
+    show_keys : :class:`bool`
+            Whether to also display the key associated with 
+            the mobject. This might be useful when debugging,
+            especially when there are a lot of mobjects in the
+            :class:`VDict`. When displayed, the key is towards
+            the left of the mobject.
+            Defaults to False
+    submob_dict : :class:`dict`
+            Is the actual python dictionary that is used to bind
+            the keys to the mobjects
+            
+    '''
+
+    def __init__(self, *pairs, show_keys=False, **kwargs):
+        if not all([isinstance(m[1], VMobject) for m in pairs ]):
```

Comment:
I'd have removed the space between `pairs` and `]`, but I also noticed that the square brackets are unnecessary - we can just use a generator expression, which is also more efficient here as it will stop checking with `isinstance` as soon as it finds a `False`.
```suggestion
        if not all(isinstance(m[1], VMobject) for m in pairs):
```

Reply:
The `VGroup` class has the square brackets so I thought to put those for similarity.

Reply:
got it